[2025-05-22 14:20:06,644](__post_init__) Проблема с подключением к серверу очереди: HTTPConnectionPool(host='10.0.5.128', port=32107): Max retries exceeded with url: /get_wplace?w=8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002439EF985E0>: Failed to establish a new connection: [WinError 10061] Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение'))
[2025-05-22 14:20:25,376](__post_init__) Проблема с подключением к серверу очереди: HTTPConnectionPool(host='10.0.5.128', port=32107): Max retries exceeded with url: /get_wplace?w=8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000237872785E0>: Failed to establish a new connection: [WinError 10061] Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение'))
[2025-05-22 14:21:19,804](__post_init__) Проблема с подключением к серверу очереди: HTTPConnectionPool(host='10.0.5.128', port=32107): Max retries exceeded with url: /get_wplace?w=8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000024D24ABF7C0>: Failed to establish a new connection: [WinError 10061] Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение'))
[2025-05-22 14:23:53,751](__post_init__) Проблема с подключением к серверу очереди: HTTPConnectionPool(host='10.0.5.128', port=32107): Max retries exceeded with url: /get_wplace?w=8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E4DBCA85E0>: Failed to establish a new connection: [WinError 10061] Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение'))
[2025-05-22 14:44:55,716](__post_init__) Проблема с подключением к серверу очереди: HTTPConnectionPool(host='10.0.5.128', port=32107): Max retries exceeded with url: /get_wplace?w=8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000296200F95E0>: Failed to establish a new connection: [WinError 10061] Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение'))
[2025-05-22 15:46:15,775](__post_init__) Проблема с подключением к серверу очереди: HTTPConnectionPool(host='10.0.5.128', port=32107): Max retries exceeded with url: /get_wplace?w=8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E5603685E0>: Failed to establish a new connection: [WinError 10061] Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение'))
[2025-05-22 17:04:07,833](__post_init__) Проблема с подключением к серверу очереди: HTTPConnectionPool(host='10.0.5.128', port=32107): Max retries exceeded with url: /get_wplace?w=8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000224375F95E0>: Failed to establish a new connection: [WinError 10060] Попытка установить соединение была безуспешной, т.к. от другого компьютера за требуемое время не получен нужный отклик, или было разорвано уже установленное соединение из-за неверного отклика уже подключенного компьютера'))
[2025-05-22 17:06:08,220](__post_init__) Проблема с подключением к серверу очереди: HTTPConnectionPool(host='10.0.5.128', port=32107): Max retries exceeded with url: /get_wplace?w=8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001C7540595E0>: Failed to establish a new connection: [WinError 10060] Попытка установить соединение была безуспешной, т.к. от другого компьютера за требуемое время не получен нужный отклик, или было разорвано уже установленное соединение из-за неверного отклика уже подключенного компьютера'))
[2025-05-22 17:06:37,863](__post_init__) Проблема с подключением к серверу очереди: HTTPConnectionPool(host='10.0.5.128', port=32107): Max retries exceeded with url: /get_wplace?w=8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001C5BC6A85E0>: Failed to establish a new connection: [WinError 10060] Попытка установить соединение была безуспешной, т.к. от другого компьютера за требуемое время не получен нужный отклик, или было разорвано уже установленное соединение из-за неверного отклика уже подключенного компьютера'))
[2025-05-22 17:08:30,511](__post_init__) Проблема с подключением к серверу очереди: HTTPConnectionPool(host='10.0.5.128', port=32107): Max retries exceeded with url: /get_wplace?w=8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002A8461A65E0>: Failed to establish a new connection: [WinError 10060] Попытка установить соединение была безуспешной, т.к. от другого компьютера за требуемое время не получен нужный отклик, или было разорвано уже установленное соединение из-за неверного отклика уже подключенного компьютера'))
[2025-05-22 17:08:52,426](__post_init__) Проблема с подключением к серверу очереди: HTTPConnectionPool(host='10.0.5.128', port=32107): Max retries exceeded with url: /get_wplace?w=8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000213AA9E95E0>: Failed to establish a new connection: [WinError 10060] Попытка установить соединение была безуспешной, т.к. от другого компьютера за требуемое время не получен нужный отклик, или было разорвано уже установленное соединение из-за неверного отклика уже подключенного компьютера'))
[2025-05-22 17:10:36,146](__post_init__) Проблема с подключением к серверу очереди: HTTPConnectionPool(host='10.0.5.128', port=32107): Max retries exceeded with url: /get_wplace?w=8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001B919C1D7C0>: Failed to establish a new connection: [WinError 10060] Попытка установить соединение была безуспешной, т.к. от другого компьютера за требуемое время не получен нужный отклик, или было разорвано уже установленное соединение из-за неверного отклика уже подключенного компьютера'))
